content/blog/2023/02/2023-02-23-gsoc2023-announcement.adoc

```diff
@@ -0,0 +1,104 @@
+
+For detailed information on Jenkins GSoC 2022, see link:/projects/gsoc/2022/[completed projects].
+
+image:/images/gsoc/2023/GSoC2023-here_we_come.png[Here we come]
+
+**What's next?**
+GSoC is link:https://opensource.googleblog.com/2023/02/mentor-organizations-announced-for.html[officially announced], and please expect more potential GSoC contributors to contact projects in our
+link:/projects/gsoc#contacts[Gitter channels and Discourse channels].
+Many communications will also happen in SIG and sub-project channels.
+We will be working hard in order to help potential participants to find interesting projects, to explore the area,
+and to prepare their project proposals before the deadline on April 4th.
+Then we will process the applications, select projects and assign mentor teams.
+
+All information about the Jenkins GSoC is available on its link:/projects/gsoc/[sub-project page].
+
+== How do I apply?
+
+See the link:/projects/gsoc/students[Information for students] page for full application guidelines.
+
+We encourage interested participants to reach out to the Jenkins community early and to start exploring project ideas.
+We also encourage participants to join the weekly link:https://docs.google.com/document/d/1UykfAHpPYtSx-r_PQIRikz2QUrX1SG-ySriz20rVmE0/edit?usp=sharing[Jenkins GSoC office hours], these meetings are set up for participants to meet org admins and mentors and to ask questions.
+Also, join our link:https://app.gitter.im/#/room/#jenkinsci_gsoc-sig:gitter.im[Gitter channel] and our
+link:https://community.jenkins.io/c/contributing/gsoc/6[Discourse server] 
+to receive information about such incoming events in the project.
+
+The application period starts on March 20th, but you should prepare now!
+Use the time before the application period to discuss and improve your project proposals.
+We also recommend that you become familiar with Jenkins and start exploring your proposal areas.
+Project ideas include quick-start guidelines and reference newbie-friendly issues
+which may help with initial study.
+If you do not see anything interesting,
+you can link:/projects/gsoc/proposing-project-ideas/[propose your own project idea].
+
+== I want to be a mentor. Is it too late?
+
+It's not!
+We are looking for more project ideas and for Jenkins contributors/users
+who are passionate about Jenkins and want to be a mentor.
+No hardcore experience required, mentors can study the project internals together with GSoC contributors and technical advisors.
+
+You can either propose a new project idea or join an existing one.
+See the link:/blog/2022/12/09/GSoC-the-gift-of-mentorship//[Call for Mentors post]
+and link:/projects/gsoc/mentors[Information for mentors] for details.
+If you want to propose a new project,
+please do so as soon as possible so that potential GSoC contributors have time to explore them and to prepare their proposals.
```

Comment:
```suggestion
If you want to propose a new project, please do so as soon as possible so that potential GSoC contributors have time to explore them and prepare their proposals.
```
small formatting/wording suggestion